core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -10,18 +10,22 @@ import scalacss.ScalaCssReact._
 
 object ReactTable {
 
-  type Model = Map[String, Any]
+  type ColumnKey = String
+  type Model = Map[ColumnKey, Any]
```

Comment:
My biggest issue with this component is this `Any` here. We could make the whole `ReactTable` a trait parameterised over a `T`, so you could have a typesafe `object ReactTableString extends ReactTable[String]...` or something like that. it's not important for now, but we can experiment with it now if you want to?